### DISCUSSION TITLE
New release please! 🙏🙏🙏

### DISCUSSION THREAD

**Discussion by moikot** [coverlet-coverage/coverlet#1619] [Q&A]
Are there any plans to release a new version with the latest patches? 
I'm really looking forward to getting it to start migrating my projects to .NET 8.0.

**Comment by Bertk** [comment]
Please use nightly build packages and send your feedback.

e.g.
https://dev.azure.com/tonerdo/coverlet/_artifacts/feed/coverlet-nightly/NuGet/coverlet.msbuild/overview/8.0.0-preview.15.g893dcc8fb6

**Comment by Bertk** [comment]
coverlet V8.0.0 release is available.
